docs/resources.rst

```diff
@@ -112,6 +115,9 @@ Solidity Tools
 * `PIET <https://piet.slock.it/>`_
     A tool to develop, audit and use Solidity smart contracts through a simple graphical interface.
 
+* `Scaffold-ETH <https://github.com/scaffold-eth/scaffold-eth>`
+    Forkable Ethereum development stack focused on fast product iterations.
+
```

Comment:
Also, looks like we already have it listed (but in the IDE section, which seems wrong).

Reply:
I removed it from the IDE section.